src/db/h2/lrsql/h2/sql/query.sql

```diff
@@ -243,9 +243,10 @@ WHERE activity_iri = :activity-iri
 -- :name query-account
 -- :command :query
 -- :result :one
--- :doc Given an account `username`, return the ID and the hashed password, which can be used to verify the account.
+-- :doc Given an account `username` or `account-id`, return the ID and the hashed password, which can be used to verify the account.
 SELECT id, passhash FROM admin_account
-WHERE username = :username
+--~ (when (:username params)   "WHERE username = :username")
+--~ (when (:account-id params) "WHERE id = :account-id")
```

Comment:
And when both are present, it performs an double check. 😈

Reply:
Is there a syntax for an OR between the two params? If so I'll use it, otherwise I don't think it matters too much.

Reply:
Yeah that was my thinking too. However from outside there isn't really a way to even get to that point, since the params validation and extraction uses `select-keys` on the body params

Reply:
To be clear I don't think this is necessarily a problem (especially since we don't even do any doubled-up queries from the app), it was just an interesting observation.